Hax/css/screen.css

```diff
@@ -328,6 +328,15 @@ p code {
   color: #fff;
 }
 
+.branding .logo-askew {
+  animation-name: normal;
+  animation-duration: .6s;
+  animation-fill-mode: forwards;
+  animation-iteration-count: 1;
+  animation-timing-function: linear;
```

Comment:
If you wanna get extra-fancy, an out-of-range cubic-bezier timing function gives it a nifty little snap when it animates. Try `cubic-bezier(.75, -2, .75, 2)` :smiley_cat: